src/stringBuffer.js

```diff
@@ -29,7 +29,21 @@
  * @return {function}
  */
 function makeBuffer() {
-  // write code here
+  let sum = '';
+
+  function buffer(string) {
```

Comment:
```suggestion
  function buffer(string = '') {
```
тогда не нужна проверка на undefined